Packs/EDL/Integrations/EDL/EDL_description.md

```diff
@@ -18,23 +22,35 @@ To access the EDL service by instance name, make sure ***Instance execute extern
- * Example: https://{cortex-xsoar_instance}/instance/execute/{EDL_instance_name}?v=panosurl&di
-6. **tr** - Whether to collapse IPs. 
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?s=10&n=50
+3. **v** - The output format. Supports `text`, `csv`, `json`, `json-seq`,`xsoar-json`, `xsoar-seq`, `mwg` and `proxysg` (alias: `bluecoat`).
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=json
+4. **q** - The query used to retrieve indicators from the system.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?q="type:ip and sourceBrand:my_source"
+5. **t** - Only with `mwg` format. The type indicated on the top of the exported list. Supports: string, applcontrol, dimension, category, ip, mediatype, number and regex.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=mwg&t=ip
+6. **sp** - Only with `text` format. If set will strip ports off URLs, otherwise will ignore URLs with ports.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=text&sp 
+7. **di** - Only with `text` format. If set will ignore URLs which are not compliant with PAN-OS URL format instead of being re-written.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=text&di
+8. **pr** - Only with `text` format. If set will strip protocols off URLs.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=text&pr
+9. **cd** - Only with `proxysg` format. The default category for the exported indicators.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=proxysg&cd=default_category
+10. **ca** - Only with `proxysg` format. The categories which will be exported. Indicators not in these categories will be classified as the default category.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=proxysg&ca=category1,category2
+11. **tr** - Whether to collapse IPs. 
     * 0 - Do not collapse. 
     * 1 - Collapse to ranges.
     * 2 - Collapse to CIDRs.
- * Example: https://{cortex-xsoar_instance}/instance/execute/{EDL_instance_name}?q="type:ip and sourceBrand:my_source"&tr=1
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?q="type:ip and sourceBrand:my_source"&tr=1
+12. **tx** - Whether to output `csv` formats as textual web pages.
+ * Example: https://{cortex-xsoar_instance}/instance/execute/{ExportIndicators_instance_name}?v=csv&tx
 
 ### When running in On-Demand mode
-Make sure you run the `!edl-update` command for the first time to initialize the export process.
+Make sure you run the `!edl-update` command for the first time to initialize the export process.
```

Comment:
we should also change this command name

Reply:
**Deprecate this command and add a copy of it with the name: `export-indicators-list-update`